packages/api-explorer/src/App.tsx

```diff
@@ -152,34 +148,35 @@ const PageLayout = styled.div<SidebarStyleProps>`
 
   ${SidebarGroup} {
     > div {
-      border-left: 1px dashed
-        ${(props) => props.theme.colors.palette.charcoal200};
-      padding: ${(props) => props.theme.space.xxsmall} 0
-        ${(props) => props.theme.space.xxsmall}
-        ${(props) => props.theme.space.xsmall};
+      border-left: 1px dashed ${({ theme }) => theme.colors.ui2};
+      padding: ${({ theme }) => `${theme.space.xxsmall} 0
+        ${theme.space.xxsmall}
+        ${theme.space.xsmall}`};
```

Comment:
Purely stylistic but one thing I've done in these kinds of cases is to de-structure a bit further to improve readability.

```suggestion
      padding: ${({ theme: { space: { xxsmall, xsmall }}}) => 
        `${xxsmall} 0 ${xxsmall} ${xsmall}`};
```